src/modules/rover_ackermann/AckermannPosControl/AckermannPosControl.cpp

```diff
@@ -61,12 +61,14 @@ void AckermannPosControl::updatePosControl()
 	if (target_waypoint_ned.isAllFinite()) {
 		float distance_to_target = (target_waypoint_ned - _curr_pos_ned).norm();
 
-		if (distance_to_target > _acceptance_radius) {
-			float arrival_speed = PX4_ISFINITE(_rover_position_setpoint.arrival_speed) ? _rover_position_setpoint.arrival_speed :
-					      0.f;
-			const float distance = arrival_speed > 0.f + FLT_EPSILON ? distance_to_target - _acceptance_radius : distance_to_target;
+		if (_arrival_speed > FLT_EPSILON) {
```

Comment:
Why is that change needed? Isn't the check below (_arrival_speed > FLT_EPSILON) enough?